test/duct/migrator/ragtime_test.clj

```diff
@@ -45,21 +45,38 @@
   (doseq [t (find-tables)]
     (jdbc/execute! db-spec [(str "DROP TABLE " (:name t))])))
 
-(deftest migration-test
+(defn- migration-works-spec [migrations-table]
+  (is (= (find-tables)
+         [{:name migrations-table}
+          {:name "foo"}
+          {:name "bar"}]))
+  (is (= @logs
+         [[::ragtime/applying ":duct.migrator.ragtime-test/create-foo#f1480e44"]
+          [::ragtime/applying ":duct.migrator.ragtime-test/create-bar#6d969ce8"]])))
+
+(defn reset-test-state []
   (reset! logs [])
-  (drop-all-tables)
-  (let [system (ig/init config)]
-    (is (= (find-tables)
-           [{:name "ragtime_migrations"}
-            {:name "foo"}
-            {:name "bar"}]))
-    (is (= @logs
-           [[::ragtime/applying ":duct.migrator.ragtime-test/create-foo#f1480e44"]
-            [::ragtime/applying ":duct.migrator.ragtime-test/create-bar#6d969ce8"]]))))
+  (drop-all-tables))
+
+(use-fixtures :each (fn [test-fn]
```

Comment:
Can you avoid use of `use-fixtures`?

Reply:
Sure I apply it. Why do you prefer to avoid them?